plugins-ui/src/modules/marketplace/models/marketplace.ts

```
@@ -0,0 +1,16 @@
+export type ViewFilter = "grid" | "list";
+
+type Plugin = {
+  id: string;
+  type: string;
+  title: string;
+  description: string;
+  metadata: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)